app/components/schema-field-radio/component.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  init() {
+    this._super(...arguments);
+    let key = this.get('key');
+    let document = this.get('document');
+    let defaultValue = this.get('property.default');
+    let initialValue =  '';
```

Comment:
The main reason is that `false` is a valid current value for a boolean property so the existing value check has to explicitly check for undefined rather than just falsey.  `document.get(key) || defaultValue || '';` would just return `''` for a boolean property.